rules/integrations/azure/defense_evasion_kubernetes_events_deleted.toml

```diff
@@ -0,0 +1,57 @@
+[metadata]
+creation_date = "2021/06/24"
+maturity = "production"
+updated_date = "2021/06/24"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies when Events are deleted in Azure Kubernetes. An adversary may delete events in Azure Kubernetes in an attempt to evade detection.
+"""
+false_positives = [
+    """
+    Events deletions may be done by a system or network administrator. Verify whether the username, hostname, and/or
+    resource name should be making changes in your environment. Events deletions from unfamiliar users or hosts
+    should be investigated. If known behavior is causing false positives, it can be exempted from the rule.
+    """,
+]
+from = "now-25m"
+index = ["filebeat-*", "logs-azure*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Azure Kubernetes Events Deleted"
+note = """## Config
+The Azure Fleet integration, Filebeat module, or similarly structured data is required to be compatible with this rule."""
+references = [
+    "https://docs.microsoft.com/en-us/azure/role-based-access-control/resource-provider-operations#microsoftkubernetes",
+]
+risk_score = 47
+rule_id = "8b64d36a-1307-4b2e-a77b-a0027e4d27c8"
+severity = "medium"
+tags = ["Elastic", "Cloud", "Azure", "Continuous Monitoring", "SecOps", "Log Auditing"]
+timestamp_override = "event.ingested"
+type = "query"
+
+query = '''
+event.dataset:azure.activitylogs and azure.activitylogs.operation_name:MICROSOFT.KUBERNETES/CONNECTEDCLUSTERS/EVENTS.K8S.IO/EVENTS/DELETE and 
+event.outcome:(Success or success)
```

Comment:
Can you provide a event around this one or relevant documentation around the Operation?